proto/minder/v1/minder.proto

```diff
@@ -31,12 +31,29 @@ enum ObjectOwner {
     reserved 1; // deprecated OBJECT_OWNER_ORGANIZATION
 }
 
+enum Relation {
+    CREATE = 0;
+    GET = 1;
+    UPDATE = 2;
+    DELETE = 3;
+    REPO_GET = 5;
+    REPO_CREATE = 6;
+    REPO_UPDATE = 7;
+    REPO_DELETE = 8;
+    ARTIFACT_GET = 9;
+    ARTIFACT_CREATE = 10;
+    ARTIFACT_UPDATE = 11;
+    ARTIFACT_DELETE = 12;
```

Comment:
What about adding annotations as such https://stackoverflow.com/questions/11474416/define-dictionary-in-protocol-buffer/11486640#11486640 to make this mapping easier?